tenant/storage_bucket.go

```diff
@@ -57,7 +57,7 @@ func (s *Store) uniqueBucketName(ctx context.Context, tx kv.Tx, oid platform.ID,
 	}
 
 	// any other error is some sort of internal server error
-	return ErrInternalServiceError(err)
+	return errors.ErrInternalServiceError(err)
```

Comment:
Similar to my question in `storage_org.go`, do you want the `errors.WithErrorOp` on the errors here?

Reply:
`uniqueBucketName` is called from `CreateBucket` and `UpdateBucket` so there is no single operation to use. It might be better to set the operation in the caller.